src/main/java/gregtech/common/items/MetaItems.java

```diff
@@ -653,7 +652,6 @@ public static void registerColors() {
         }
     }
 
-    @SubscribeEvent
```

Comment:
It was not acting as a subscribe event where it was (and instead called by the actual subscribe event in the client proxy), so the annotation was removed to prevent confusion.